improver_tests/acceptance/SHA256SUMS

```diff
@@ -79,6 +79,9 @@ f3848563a1ff89a06e487036389c49849d345b7d1cab8e6ac3a961ecf9c879c6  ./combine/boun
 e88c3624b3b290db8c8203ca8802f2d8f341da0879bebaec5bb4570aa54bc27c  ./combine/broadcast/input_larger_cube.nc
 b5ca2030a23ba6440c712a659f6f4f3dcbe8566afeef1a68e2e5a6a9b6e7484e  ./combine/broadcast/input_smaller_cube.nc
 02a9f7a2fe3e80d9e7d43e87e7c8031f6e82be2137a02cd9cfe8c93f79906a39  ./combine/broadcast/kgo.nc
+e7a0d4e2c2541dfea51a71d0689ba100290de8986d79f5d858101f26c77cd741  ./combine/cellmethods/kgo.nc
```

Comment:
These files contain some unexpected attributes i.e. `NCO` and `history`, which cause warnings in the interpret_metadata CLI.

Reply:
Thanks for picking that up. I hadn't appreciated that NCO tools add history attributes when working on the netcdf files, though it makes sense. I've removed the extraneous attributes from the input accumulation and regenerated the KGO.